src/lib/marineCli.ts

```diff
@@ -62,16 +63,28 @@ export const initMarineCli = async (
     cwd,
     printOutput = true,
   }): Promise<string> => {
-    return execPromise({
-      command: marineCLIPath,
-      args,
-      flags,
-      spinnerMessage:
-        message === undefined
-          ? undefined
-          : getMessageWithKeyValuePairs(message, keyValuePairs),
-      options: { cwd },
-      printOutput,
-    });
+    try {
+      return await execPromise({
+        command: marineCLIPath,
+        args,
+        flags,
+        spinnerMessage:
+          message === undefined
+            ? undefined
+            : getMessageWithKeyValuePairs(message, keyValuePairs),
```

Comment:
Let's move that to a separate variable plz? in-place takes 4 lines, making it hard to grasp